generated/proto/placementpb/placement.proto

```diff
@@ -41,7 +41,7 @@ message Placement {
 
 message Instance {
   string id = 1;
-  string rack = 2;
+  string isolation_group = 2;
```

Comment:
I think this is backward compatible, but might be worth double checking.

Reply:
Yeah verified this is backward compatible